lxdock/conf/config.py

```diff
@@ -115,6 +153,27 @@ def _get_container_config_dict(self, container_dict):
         del container_config['containers']
         return container_config
 
+    def _get_dotenv_items(self, env_filepath):
```

Comment:
Yeah, that seems tricky. Maybe we could simply add a `printenv` call at the end of the `.env` script and parse *that* instead. `printenv` would give us something that is easier to parse.